dashboard/src/actions/catalog.ts

```diff
@@ -225,8 +225,8 @@ export function checkCatalogInstalled(): ThunkAction<
   ServiceCatalogAction
 > {
   return async dispatch => {
-    const isInstalled = await ServiceCatalog.isCatalogInstalled();
-    isInstalled ? dispatch(installed()) : dispatch(notInstalled());
-    return isInstalled;
+    const isServiceCatalogInstalled = await ServiceCatalog.isCatalogInstalled();
```

Comment:
I got confused when I was reading the component `ServiceInstanceList`. There I didn't know what "was installed" until I read the ternary operator `isInstalled ? ... : <ServiceCatalogNotFound />`. I think it's useful to have it verbose there (and in this file it doesn't hurt either), don't you think?

Reply:
Fair enough, I almost wonder if it would make more sense if it was `apiExists` or something?

Reply:
that would have the same problem, to be self explanatory it should be something like `serviceCatalogAPIExists`, since the action is `installed` or `notInstalled` I would leave it as it is to avoid more chained changes.